Automattic/wp-calypso
# Show commission fee at Earn page

Adding mention of commission + Stripe fee at Earn page on couple places.

- Related to this, see also aligning Jetpack plans in the backend D112347-code
- Copy isn't the best, but I used what already existed so that I don't have to wait for translations. Next up I'll improve the copy.
- I linked to the support doc where commissions across plans is listed. We should link to plans grid from the docs (TODO)
    <img width="1323" alt="Screenshot 2023-06-01 at 15 00 02" src="https://github.com/Automattic/wp-calypso/assets/87168/1600cc30-1b80-4742-886e-4a1fee5674d0">

- Plans grid doesn't have the commission shown across plans yet; once that's there let's swap link to plans grid instead of docs. That way this works as an upsell to lower the fee. Copy could then be for example:

    > With your current plan, the transaction fee for payments is 10% (+ Stripe fees). Upgrade to lower it.

## Proposed Changes

* Adds blurb about commission fee at Earn page. When the page will be redesigned, we'll include this info then too.

## Testing Instructions

* You should have Stripe disconnected, start with free site
* Go to /earn
* See fees for your plan — try updating to a plan and refresh. You should see fees also for Jetpack sites.
 
<img width="1130" alt="Screenshot 2023-06-02 at 13 11 54" src="https://github.com/Automattic/wp-calypso/assets/87168/1b3915f1-9d83-4d9d-8cc5-abf087055cfb">

* Press "Enable payments button" and see Stripe connection intro, note fees again

<img width="1219" alt="Screenshot 2023-06-02 at 12 38 07" src="https://github.com/Automattic/wp-calypso/assets/87168/8a57a985-36d1-4e53-ab3f-b3763f543dcd">

* After connecting to Stripe, you'll see fees again. This was already here so no changes to this.

<img width="1155" alt="Screenshot 2023-06-02 at 13 09 37" src="https://github.com/Automattic/wp-calypso/assets/87168/c581746a-5f47-43f3-b271-2c9470a3640a">


## Pre-merge Checklist
